datastore-v1-proto-client/src/main/resources/META-INF/native-image/native-image.properties

```diff
@@ -0,0 +1 @@
+Args = --enable-url-protocols=https,http
```

Comment:
Does this library use HTTP (not HTTPS) somewhere in users' environment? (What's the error without this argument?)

If the request is only for testing, then this file should be under test.

Reply:
Looking at the stacktrace, this is coming from `com.google.datastore.v1.client.RemoteRpc`: 
```
 Caused by: java.lang.IllegalArgumentException: java.net.MalformedURLException: Accessing an URL protocol that was not enabled. The URL protocol https is supported but not enabled by default. It must be enabled by adding the --enable-url-protocols=https option to the native-image command.
       com.google.api.client.http.GenericUrl.parseURL(GenericUrl.java:679)
       com.google.api.client.http.GenericUrl.<init>(GenericUrl.java:125)
       com.google.api.client.http.GenericUrl.<init>(GenericUrl.java:108)
       com.google.datastore.v1.client.RemoteRpc.resolveURL(RemoteRpc.java:161)
       com.google.datastore.v1.client.RemoteRpc.<init>(RemoteRpc.java:62)
```
Normally this configuration would come from gax, but this module doesn't import it so we had to add this setting in.

Reply:
The error is coming from `RemoteRpc.resolveURL` but it's used in [`RemoteRpc#call` ]( https://github.com/googleapis/java-datastore/blob/957a6b5b940edc476344a38cc6038a4a4851039f/datastore-v1-proto-client/src/main/java/com/google/datastore/v1/client/RemoteRpc.java#L86) which is later referenced in `Datastore`. For this reason, I think we would still need it in the main configuration?